slides/2023-03-dpg/gammapy_unfolding_nbiederbeck/README.md

```diff
@@ -0,0 +1,7 @@
+# Unfolding the Crab Nebula Flux with Gammapy
```

Comment:
I have a naive question: this unfolding is made is the reco energy? 
If yes, the comparison with fitting is not strictly correct as a fit return a spectra in true energy...

Reply:
The unfolding results are differential flux points binned in true energy